main.css

```diff
@@ -9,77 +9,131 @@ h1 {
 
 
 form {
-/* Just to center the form on the page */
-margin: 0 auto;
-width: 400px;
-/* To see the outline of the form */
-padding: 1em;
-border: 3px solid aqua;
-border-radius: 1em;
+    /* Just to center the form on the page */
+    margin: 0 auto;
+    width: 500px;
+    /* To see the outline of the form */
+    padding: 1em;
+    border: 3px solid aqua;
+    border-radius: 1em;
+
 
+}
 
+label {
+    /* labels display on top of field. */
+    display: block;
 }
 
 .print {
-margin:  auto;
-width: 400px;
-padding: 1em;
-border-radius: 1em;
-background-color: aquamarine;
+    /* this is the div where user input prints after submit */
+    margin: auto;
+    width: 500px;
+    padding: 1em;
+    border-radius: 1em;
+    background-color: aquamarine;
 }
 
 
-form div + div {
-margin-top: 1em;
+.wrapper {
+    /* this is the outer div */
+    margin-top: 1em;
 }
 
-label {
-/* To make sure that all labels have the same size and are properly aligned */
-display: inline-block;
-width: 90px;
-text-align: right;
+.container_user_name {
+    /* contains first and last name */
+    display: inline-block;
+    padding: 1em;
```

Comment:
it's a bit confusing to have the `wrapper` class and the `container_[something]` classes both apply vertical spacing. since the `wrapper` creates a block-level element, it should be responsible for vertical spacing while the class for the child elements should control horizontal spacing. i also recommend using `margin` to create that horizontal spacing since it is setting the space between elements. `padding` is more appropriate to create spacing between an element's border and its content